README.md

```diff
@@ -11,3 +11,29 @@ Do not push any code to that repository.
 3. Make sure this repository have the proper `APP_PRIVATE_KEY` secret set
 4. Make the `nextcloud_release_service` user is a co-maintainer of your app on https://apps.nextcloud.com/
 5. Make sure you have admin rights to this repository
+
+# Checklist for the pull request
+
+## Release pull request
+
+- [ ] Pull latest stable branch `git checkout stableX`
+- [ ] Branch off a release branch `git checkout -b release/X.X.X`
+- [ ] Bump versions
+  - [ ] appinfo/info.xml
+  - [ ] package.json
+  - [ ] package-lock.json
+- [ ] Update changelog
+- [ ] Commit the version bump and push to origin 
+- [ ] Merge release pull request
+
+## Release the app
+
+- [ ] Once merged, checkout the stable branch again `git checkout stableX` and `git pull`
```

Comment:
We might want to adjust to master since we do not have any stable branches for now